Clicker/SectionControllers/GroupControlsSectionControllers/PollSettingsSectionController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  PollSettingsSectionController.swift
+//  Clicker
+//
+//  Created by Mathew Scullin on 3/12/19.
+//  Copyright © 2019 CornellAppDev. All rights reserved.
+//
+
+import IGListKit
+
+class PollSettingsSectionController: ListSectionController {
+    
+    // MARK: - Data vars
+    var settingsModel: PollsSettingModel!
+    
+    // MARK: - Constants
+    let cellHeight: CGFloat = 96
+    let horizontalPadding: CGFloat = 16
+    let verticalPadding: CGFloat = 16
+    
+    override init() {
+        super.init()
+        self.inset = UIEdgeInsets(top: verticalPadding, left: 0, bottom: 0, right: 0)
+    }
+    
+    // MARK: - ListSectionController overrides
+    override func sizeForItem(at index: Int) -> CGSize {
+        guard let containerSize = collectionContext?.containerSize else {
+            return .zero
+        }
+        return CGSize(width: containerSize.width - (2 * horizontalPadding), height: cellHeight)
+    }
+    
+    override func cellForItem(at index: Int) -> UICollectionViewCell {
+        let cell = collectionContext?.dequeueReusableCell(of: SettingCell.self, for: self, at: index) as! SettingCell
+        cell.configure(for: settingsModel)
+        cell.layer.cornerRadius = 5
```

Comment:
Can we set the cornerRadius inside the SettingCell? Is there a reason why it has to be specified in cellForItem?

Reply:
+1

Reply:
for some reason when doing it inside the cell, it wasn't working